uwblueprint/ccbc
# [Feat] Search Box Component

## Notion ticket link

[Search Function: Search Box Component](https://www.notion.so/uwblueprintexecs/Search-Function-Search-Box-Component-107a6b6ffc8a47a18e19b5e07617bfea)



## Implementation description
* Created Search Box component using Chakra's input component and icons



## Steps to test
1. Type into input field
2. Click Search to ensure cursor changes
3. Click left icon to ensure cursor doesn't change
4. Check styling for focus and hover

## Checklist
